third_party/move/tools/move-package/src/lib.rs-warnings-are-errors

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
Is this a leftover file? We should remove it.

Reply:
Oops.